Sources/PaymentsCore/Networking/Models/Responses/AccessTokenResponse.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+public class AccessTokenResponse: Codable {
```

Comment:
Thoughts on making this a struct vs a class?

Reply:
struct!

Reply:
They're good for modeling data due to it's "value-type" nature